src/orchestrator-process/src/lib.rs

```diff
@@ -181,9 +181,14 @@ impl NamespacedOrchestrator for NamespacedProcessOrchestrator {
             port_metadata_file_locations[i] = Some(port_metadata_file_location.clone());
 
             if pid_file_location.exists() && port_metadata_file_location.exists() {
+                if let None = &system {
+                    system = Some(sysinfo::System::new_with_specifics(
+                        RefreshKind::new().with_processes(Default::default()),
+                    ));
+                }
```

Comment:
Can you do something like this to avoid the unwrap?

```suggestion
                let system = system.get_or_insert_with(|| sysinfo::System::new_with_specifics(
                        RefreshKind::new().with_processes(Default::default()),
                    ));
```

Reply:
D'oh it seems I'm not writing enough code these days :) Thanks!